config/config.yaml

```diff
@@ -9,7 +9,7 @@ Burress: Files/Previous Fish Measurements - Burress et al. 2016.csv
 Burress_DOI: doi:10.5072/FK2/KLN3CS
 
 # Input settings
-limit_images: 20 #if "" will run all, otherwise must input in integer
+limit_images: 20 #if "" will run all images, otherwise must input in integer to limit number of images
```

Comment:
This is good, but it would be helpful to mention this in the README so users would know to change this file if they want to reproduce with all the data.

Reply:
Great! changed it on main.